### ISSUE TITLE
Jittery Battery levels

### ISSUE THREAD

**Issue by sidharthv96** [jariz/Akku#65]
The battery level seems to jump around a lot sometimes, is it possible to apply a smoothening function or to filter out the noisy data ?

**Comment by SpyGuy0215** [comment]
Yes, it stays relatively normal but jitters (presumably when updating/refreshing) the battery level

**Comment by nighto** [comment]
+1, over here with an Aukey EP-T21 it sometimes keeps jumping between the correct percentage and 100%.
